<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="fr" sourcelanguage="en">
  <context>
    <name>Preset</name>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="223"/>
      <source>Create Cue</source>
      <translation>Créer un go</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="224"/>
      <source>Load on selected Cues</source>
      <translation>Charger les cues sélectionnés</translation>
    </message>
  </context>
  <context>
    <name>Presets</name>
    <message>
      <location filename="../../../plugins/presets/presets.py" line="67"/>
      <source>Presets</source>
      <translation>Pré-réglages</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets.py" line="72"/>
      <source>Save as preset</source>
      <translation>Sauvegarder un pré-réglage sous</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="67"/>
      <source>Cannot scan presets</source>
      <translation>Impossible d&apos;examiner les pré-réglages</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="71"/>
      <source>Error while deleting preset &quot;{}&quot;</source>
      <translation>Erreur lors de la suppression du pré-réglage &quot;{}&quot;</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="78"/>
      <source>Cannot load preset &quot;{}&quot;</source>
      <translation>Impossible de charger le pré-réglage &quot;{}&quot;</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="84"/>
      <source>Cannot save preset &quot;{}&quot;</source>
      <translation>Impossible de sauvegarder le pré-réglage &quot;{}&quot;</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="90"/>
      <source>Cannot rename preset &quot;{}&quot;</source>
      <translation>Impossible de renommer le pré-réglage &quot;{}&quot;</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="101"/>
      <source>Select Preset</source>
      <translation>Sélectionner le pré-réglage</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="423"/>
      <source>Preset name</source>
      <translation>Nom du pré-réglage</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="219"/>
      <source>Add</source>
      <translation>Ajouter</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="220"/>
      <source>Rename</source>
      <translation>Renommer</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="221"/>
      <source>Edit</source>
      <translation>Éditer</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="222"/>
      <source>Remove</source>
      <translation>Supprimer</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="227"/>
      <source>Export selected</source>
      <translation>Exporter le sélectionné</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="230"/>
      <source>Import</source>
      <translation>Importer</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="309"/>
      <source>Warning</source>
      <translation>Attention</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="268"/>
      <source>The same name is already used!</source>
      <translation>Le même nom est déjà utilisé !</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="344"/>
      <source>Cannot export correctly.</source>
      <translation>Impossible d&apos;exporter correctement</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="369"/>
      <source>Cannot import correctly.</source>
      <translation>Impossible d&apos;importer correctement.</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="424"/>
      <source>Cue type</source>
      <translation>Type de cue</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets.py" line="72"/>
      <source>Load on cue</source>
      <translation>Charger au go</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets.py" line="72"/>
      <source>Load on selected cues</source>
      <translation>Charger les go sélectionnés</translation>
    </message>
    <message>
      <location filename="../../../plugins/presets/presets_ui.py" line="309"/>
      <source>Cannot create a cue from this preset: {}</source>
      <translation>Impossible de créer un go depuis ce pré-réglage : {}</translation>
    </message>
  </context>
</TS>
